Source/UI/Validator/AddressLine1Validator.swift

```diff
@@ -1,8 +1,8 @@
 import Foundation
 
 class AddressLine1Validator: Validator {
-    func validate(text: String?) -> Bool {
-        isEmpty(text: text)
+    func isInvalid(value: Any?) -> Bool {
+        isEmpty(text: value as? String)
```

Comment:
I cannot get behind this naming. It reads a lot cleaner `isValid`.

Additionally, we cannot have Any as parameter, it just strips away any type safety.  A different approach is needed.